docs/data-sources/current_role.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "snowflake_current_role Data Source - terraform-provider-snowflake"
+subcategory: ""
+description: |-
+  
+---
+
+# snowflake_current_role (Data Source)
+
+
+## Example Usage
+
+```terraform
+data "snowflake_current_role" "this" {}
+
+resource "aws_ssm_parameter" "snowflake_account_role" {
+  name  = "/snowflake/account_role"
+  type  = "String"
+  value = data.snowflake_current_account.this.current_role
```

Comment:
should be udpated to `data.snowflake_current_account.this.name`

Reply:
Yess, on it!

Reply:
Done